### PR TITLE
Updated docs for 1.1/1.1.1.

### PR DESCRIPTION
**Pull request by arthurevans**
@ebidel @robdodson PTAL.

Note in addition to doc updates, I updated the samples bower.json file, so we're using newer versions of google-map and firebase-element.

### CODE REVIEW

File: 1.0/docs/devguide/styling.md [hunk 1]
```diff
@@ -557,9 +572,63 @@ Polymer elements (for example, in `<style>` elements within a custom element's
...
+
+To use a style module in an element:
+
+    <!-- import the module  -->
+    <link rel="import" href="../shared-styles/shared-styles.html">
+    <dom-module id="x-foo">
+      <template>
+        <!-- include the style module by name -->
+        <style include="shared-styles"></style>
+        <style>:host { display: block; }</style>
```
Comment (ebidel) [inline 1.0/docs/devguide/styling.md]: you can also combine these: 

`<style include="shared-styles">:host { display: block; }</style>`

**Comment by ebidel** [comment]
LGTM. Just the one comment

File: 1.0/docs/devguide/styling.md (hunk 1 above)
Comment (arthurevans) [inline 1.0/docs/devguide/styling.md]: Do the included styles go before or after the styles declared in the body of the element?

I'm guessing before...? Anyone know?

Comment (ebidel) [inline 1.0/docs/devguide/styling.md]: Good question. Don't know.

**Comment by robdodson** [comment]
I'm going to merge, I think @arthurevans has already pushed these changes live and I need to push another update and want to make sure I stay in sync

**Pull request merged**